packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -471,10 +471,10 @@ export const ReactEditor = {
     editor: ReactEditor,
     domPoint: DOMPoint,
     options: {
-      exactMatch: T
-      suppressThrow: T
+      exactMatch: boolean
+      suppressThrow: boolean
     }
-  ): T extends true ? Point | null : Point {
+  ): Point | null {
```

Comment:
Not a huge fan of the signature change, it makes sense that it returns a non-nullable value when not using `suppressThrow` (although the current definition is wrong as `exactMatch` should be boolean and not `T` since it doesn't influence the 'nullability' of the return, as far as I can see). The same is the case for `toSlateRange`.